src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,132 @@
+package wolox.training.models;
```

Comment:
add the missing default contructor, it's necessary for jpa and hibernate mapping
`public Book(){}`